src/Api/ItemDataProviderInterface.php

```diff
@@ -31,6 +30,7 @@
      *
      * @throws ResourceClassNotSupportedException
      * @throws InvalidArgumentException
+     * @throws ExceptionInterface
```

Comment:
So you can remove the other `throws` as the `ResourceClassNotSupportedException` and `InvalidArgumentException` exceptions implements the `ExceptionInterface`.